src/geometry/euclidean/triangle.lean

```diff
@@ -154,19 +155,19 @@ begin
                 ∥x∥ * ∥y∥ * ∥x - y∥ * ∥x - y∥ =
               (real.sin (angle x (x - y)) * (∥x∥ * ∥x - y∥)) *
                 (real.sin (angle y (y - x)) * (∥y∥ * ∥x - y∥)), { ring },
-    have H2 : inner x x * (inner x x - inner x y - (inner x y - inner y y)) -
+    have H2 : ⟪x, x⟫ * (inner x x - inner x y - (inner x y - inner y y)) -
                 (inner x x - inner x y) * (inner x x - inner x y) =
               inner x x * inner y y - inner x y * inner x y, { ring },
-    have H3 : inner y y * (inner y y - inner x y - (inner x y - inner x x)) -
+    have H3 : ⟪y, y⟫ * (inner y y - inner x y - (inner x y - inner x x)) -
                 (inner y y - inner x y) * (inner y y - inner x y) =
               inner x x * inner y y - inner x y * inner x y, { ring },
```

Comment:
If using the new notation in the geometry files, I'd expect it to be used consistently everywhere in those files. Not, as here, only for a few particular instances of the inner product, omitting even others within the same expression.

Reply:
Yeah, that was lazy on my part -- I'll change the rest to the new notation as well.